### PR TITLE
jerry: support %TypedArray%.prototype.slice

### PR DESCRIPTION
**Pull request by Txiaozhe**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by Txiaozhe** [comment]
#479

File: deps/jerry/jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c [hunk 1]
```diff
@@ -768,9 +769,130 @@ ecma_builtin_typedarray_prototype_set (ecma_value_t this_arg, /**< this argument
...
  */
+static ecma_value_t
+ecma_builtin_typedarray_prototype_object_slice (ecma_value_t this_arg, /**< 'this' argument */
+                                                ecma_value_t arg1, /**< start */
+                                                ecma_value_t arg2) /**< end */
+{
+  ecma_value_t ret_value = ECMA_VALUE_EMPTY;
+
+  /* 1. */
+  ECMA_TRY_CATCH (obj_this,
```
[the remark this answers is not in the document]
Comment (Txiaozhe) [reply deps/jerry/jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c]: Thanks, I think the Array and TypedArray share the same algorithm, so I reuse the function of array, may it not very rigorous, but jerry does.

**Comment by Txiaozhe** [comment]
Is this checking smoothly? @legendecas

**Comment by legendecas** [comment]
> Is this checking smoothly? @legendecas
There might be something wrong with travis. A new trigger might be required to re-run the CI.

**Comment by Txiaozhe** [comment]
> > Is this checking smoothly? @legendecas
> 
> There might be something wrong with travis. A new push might be required to re-run the CI.

Yes, thanks! Maybe I will rebuild this implementation :)

**Pull request closed without merging**